### PR TITLE
feat(dapp-connector): Add CIP-95 support

### PR DESCRIPTION
**Pull request by michalszorad**
## Description / Change(s) / Related issue(s)
Add CIP-95 support
##### Ticket

[YOMO-1193](https://emurgo.atlassian.net/browse/YOMO-1193)

[YOMO-1193]: https://emurgo.atlassian.net/browse/YOMO-1193?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by banklesss — APPROVED** [review]

File: apps/wallet-mobile/src/yoroi-wallets/cardano/constants/cardano-config.ts [hunk 1]
```diff
@@ -62,6 +62,7 @@ export const cardanoConfig = freeze(
             roles: {
               external: 0,
               internal: 1,
+              drep: 3,
```
Comment (stackchain) [inline apps/wallet-mobile/src/yoroi-wallets/cardano/constants/cardano-config.ts]: @michaeljscript byron doesn't support staking, so it won't support governance, if this if for typing only, by checking if is 1852 will make TS infer the type.

**Review by stackchain — APPROVED** [review]
lgtm, 1 point only

File: apps/wallet-mobile/src/yoroi-wallets/cardano/constants/cardano-config.ts (hunk 1 above)
Comment (michalszorad) [reply apps/wallet-mobile/src/yoroi-wallets/cardano/constants/cardano-config.ts]: Many thanks, I'll update

**Comment by michalszorad** [comment]
WIP - byron wallet needs additional checks for CIP95

File: apps/wallet-mobile/src/yoroi-wallets/cardano/constants/cardano-config.ts (hunk 1 above)
Comment (michalszorad) [reply apps/wallet-mobile/src/yoroi-wallets/cardano/constants/cardano-config.ts]: That's now updated

**Review by stackchain — APPROVED** [review]

**Pull request merged**